### ISSUE TITLE
can not reach your PSNR?

### ISSUE THREAD

**Issue by zymize** [Zheng222/IDN-tensorflow#6]

**Comment by zymize** [comment]
hello, I reproduced your program,but I cannot reach your PSNR. I use2080ti,windows 10.
 thanks

**Comment by Zheng222** [comment]
Hello, I recommend that you train a few more epochs at learning rates of 2e-4 and 1e-4.

**Comment by zymize** [comment]
Thank you very much.
There is another question that I would like to consult .
In your distillation model,there is two lines of code:

"tmp = tf.layers.conv2d(x, 48, (3, 3), (1, 1), padding='same', activation=lrelu, name=name+'/conv1')"

 "tmp2 = tf.layers.conv2d(tmp2, 80, (3, 3), (1, 1), padding='same', activation=lrelu, name=name+'/conv6')"
How to confirm 48 and 80 filters ? is this a result of continuous experimention?

**Comment by zymize** [comment]
Thank you very much.
